picard/formats/id3.py

```diff
@@ -494,6 +494,19 @@ def _get_tags(self, filename):
         except mutagen.id3.ID3NoHeaderError:
             return compatid3.CompatID3()
 
+    def _load_preserved_config(self, metadata):
+        # Adding additional config to metadata to prevent unecessary saving
+        metadata['~config:rating_steps'] = config.setting['rating_steps']
+        metadata['~config:rating_user_email'] = config.setting['rating_user_email']
```

Comment:
Also currently in the above code, it only prevents writing metadata to files. I haven't implemented prevent_save for other operations.(Saving image to files, rename files, move files, move additional files, delete empty dirs)

I can include it though.